scripts/calculateKpi/protocols/tetherV0/parseReferralTag/getUserOperations.ts

```diff
@@ -0,0 +1,141 @@
+import {
```

Comment:
this is copied from https://github.com/divvi-xyz/divvi-relayer/blob/main/src/endpoints/submitReferral/getUserOperations.ts